### PR TITLE
Fix #91 - Added elapsed time for Init, Deploy & Undeploy commands

### LINKED ISSUE #91: Log total time for running tye init and tye deploy
Like dotnet build, printing the total runtime would be good for diagnostics.

### PR DESCRIPTION
**Pull request by NileshGule**
Added the elapsed time functionality to Init, Deploy and Undeploy commands. The time displayed is similar to the build output command 
tye undeploy
Tye Deploy

### CODE REVIEW

File: codealike.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"projectId":"cc024380-9dc5-11ea-bbbe-0905758693ce","projectName":"tye"}
```
Comment (jkotalik) [inline codealike.json:1]: What's this file for?

Comment (NileshGule) [reply codealike.json:1]: That file is related to the code analysis tool I use personally. I had added it to the list in the .gitignore file.

Comment (davidfowl) [reply codealike.json:1]: Can you remove this file?

Comment (NileshGule) [reply codealike.json:1]: @davidfowl I have removed the file reference from .gitignore

**Comment by jkotalik** [comment]
Thanks!

**Pull request merged**